.gitignore

```diff
@@ -19,6 +19,7 @@
 /keys
 /nbproject
 /node_modules
+/tags
```

Comment:
It's a Ctags file, so that text editors like vim can understand syntax of the language and provide features like "go to definition". Merging simply for future convenience, otherwise I need to remember to delete this file before pushing changes :)